bangazonapi/views/paymenttype.py

```diff
@@ -34,8 +34,8 @@ def create(self, request):
         new_payment = Payment()
         new_payment.merchant_name = request.data["merchant_name"]
         new_payment.account_number = request.data["account_number"]
-        new_payment.expiration_date = request.data["create_date"]
-        new_payment.create_date = request.data["expiration_date"]
+        new_payment.expiration_date = request.data["expiration_date"]
```

Comment:
This fix is for a different ticket. Next time make sure you're only doing 1 ticket per pr. (and close out #19 since this is the right solution)

Reply:
Oh okay. I was playing with it last night when looking through the tickets and totally forgot to make a separate PR for it last night. Apologies.